mediacapture-streams/MediaDevices-enumerateDevices-returned-objects.https.html

```diff
@@ -14,7 +14,7 @@
 {
     promise_test(async () => {
         if (callGetUserMedia) {
-            await setMediaPermission();
+            await setMediaPermission("granted");
```

Comment:
Why this change? Isn't `"granted"` already the default? Whichever way we go we should be consistent.

Reply:
reverted in de937849ae